### DISCUSSION TITLE
Is it possible to have reversed cards only?

### DISCUSSION THREAD

**Discussion by paulrouget** [st3v3nmw/obsidian-spaced-repetition#801] [Q&A]
I'm using this format:

CHINESE ::: ENGLISH

I want to see the reverse cards only, but it says there's 2 cards:

CHINESE / ENGLISH
and
ENGLISH / CHINESE.

I'm only interested in ENGLISH / CHINESE.

**Comment by ronzulu** [comment]
Hi @paulrouget, not at present. There is an existing request to add this functionality:

[FEAT] Choose A or B or random side per review #632 

Cheers
Ronny
